storage/rocksdb/ha_rocksdb.cc

```diff
@@ -9511,24 +9837,25 @@ int ha_rocksdb::check_and_lock_unique_pk(const uint key_id,
     2) T1 Get(empty) -> T1 Put(insert, not committed yet) -> T2 Get(empty)
        -> T2 Put(insert, blocked) -> T1 commit -> T2 commit(overwrite)
   */
-  const rocksdb::Status s = get_for_update(
-      row_info.tx, *m_pk_descr, row_info.new_pk_slice, &m_retrieved_record);
+  const rocksdb::Status s =
```

Comment:
Not sure of the size of rocksdb::Status, perhaps slightly more efficient to
```suggestion
  const rocksdb::Status &s =
```

Reply:
I think that compiler will optimize it.
BTW, there are 29 similar matches across 6 files in MyRocks.